src/Marten.Testing/ConnectionSource.cs

```diff
@@ -5,7 +5,7 @@ namespace Marten.Testing
     public class ConnectionSource : ConnectionFactory
     {
         public static readonly string ConnectionString = Environment.GetEnvironmentVariable("marten_testing_database")
-            ?? "Host=localhost;Port=5432;Database=marten;Username=postgres;password=postgres";
+            ?? "Host=localhost;Port=5432;Database=postgres;Username=postgres;password=postgres";
```

Comment:
Gah, sorry!

Reply:
That was also happening to me quite often until I set up the env variable ;)